src/xcode/ENA/ENA/Resources/Localization/de.lproj/privacy-policy.html

```diff
@@ -203,14 +203,22 @@ <h2>
 			2. Ist die Nutzung der App freiwillig?
 		</h2>
 		<p>
-			Die Nutzung der App ist freiwillig. Es ist allein Ihre Entscheidung, ob Sie die App installieren, welche App-Funktionen Sie nutzen und ob Sie Daten mit anderen teilen. Alle App-Funktionen, die eine Weitergabe Ihrer personenbezogenen Daten an das RKI oder an andere Nutzer erfordern, holen vorher Ihr ausdrückliches Einverständnis ein. Falls Sie ein Einverständnis nicht erteilen oder nachträglich zurücknehmen, entstehen Ihnen keine Nachteile.
-			
+			Die Nutzung der App ist freiwillig. Es ist allein Ihre Entscheidung, ob Sie die App installieren, welche App-Funktionen Sie nutzen und ob Sie Daten mit anderen teilen. Grundsätzlich holen die Haupt-Funktionen der App, die eine Weitergabe Ihrer personenbezogenen Daten an das RKI oder an andere Nutzer erfordern, vorher Ihr ausdrückliches Einverständnis ein. Im Zusammenhang mit den offiziellen digitalen COVID-19-Zertifikaten (COVID-19-Testzertifikat und COVID-19-Impfzertifikat) gibt es neue gesetzliche Grundlagen für die Erstellung der Nachweise. In diesem Zusammenhang ist daher keine zusätzliche Einwilligung erforderlich. Die Zertifikate werden aber nur erstellt, wenn Sie dies wünschen. Die Anforderung und Verwendung der digitalen COVID-19-Zertifikate ist freiwillig.
```

Comment:
please remove the number 19 from "COVID-19-Testzertifikat" and "COVID-19-Impfzertifikat".